tests/installation/opensuse_welcome.pm

```diff
@@ -15,7 +15,17 @@ use utils;
 use x11utils 'handle_welcome_screen';
 
 sub run {
-    handle_welcome_screen;
+    # In case of upgrade scenario, check if opensuse_welcome window has been already deactivated from startup
+    if (get_var('UPGRADE')) {
+        my @tags = qw(generic-desktop opensuse-welcome);
+        push(@tags, qw(gnome-activities opensuse-welcome-gnome40-activities)) if check_var('DESKTOP', 'gnome');
+        assert_screen \@tags;
```

Comment:
Theoretically, this is open for a race - not sure if in reality this might be a problem: Think 'desktop paints', needle matches 'generic-desktop', and (delayed) opensuse-welcome starts

Reply:
That's what I mentioned in the previous PR as well, but it does not appear to be an issue in practice

Reply:
ok - let's observe it

Reply:
Till now I also haven't seen any issue during the multiple VRs. In case that failures appear in the future, we can go with the checking of `config/autostart`.